src/components/Page/PageTitle.tsx

```diff
@@ -6,7 +6,8 @@ export interface IPageTitleProps {
 }
 
 export const PageTitle: React.FC<IPageTitleProps> = ({ title }) => {
-  const newTitle = title ? `${appTitle} | ${title}` : appTitle;
+  const newTitle =
+    title && title !== appTitle ? `${appTitle} | ${title}` : appTitle;
```

Comment:
Generally speaking, probably, moving check into `useEffect` as separate `if .. else` will be better readable.

But here I don't think it would be that beneficial, so we may leave it as it is now.